config/initializers/i18n.rb

```diff
@@ -0,0 +1,6 @@
+Rails.application.configure do
+  # Settings specified here will take precedence over those in config/application.rb.
```

Comment:
I'm pretty sure it's this way. I couldn't find any site telling that initializers take precedence over application.rb, and that's what I've test it myself. What I've done is override the `config.i18n.load_path` (defined at the initializer) in the `application.rb` adding the following code to the Application class definition

```ruby
Rails.application.configure do
  config.i18n.load_path = Dir[]
end
```

And when the app loaded, it still figured out where the files for the i18n were.

Having proved this, and knowing that the initialization process is always the same way (expect manual changes, of course), is what makes me believe what I've stated.

What do you think about this?